src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+
+import { getUser } from '../../utils';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  addTodo: (todo: Todo) => void;
+  users: User[];
+}
+
+export const TodoForm: React.FC<Props> = ({ addTodo, users }) => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasUserError, setHasUserError] = useState(false);
+
+  const handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setHasTitleError(false);
+  };
+
+  const handleChangeUser = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setHasUserError(false);
+  };
+
+  const restart = () => {
+    setTitle('');
+    setUserId(0);
+    setHasUserError(false);
+    setHasTitleError(false);
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (userId === 0) {
+      setHasUserError(true);
+    }
```

Comment:
```suggestion
    if (userId === 0) {
      setHasUserError(true);
      
      return;
    }
```

Reply:
we add return in this if brunch to make our method not submit if there is userError

Reply:
I did like you suggested, but in this case the error message is shown only for field in the first conditional statement even if two fields are empty / not selected. maybe I'm missing something